app/graph/page.tsx

```diff
@@ -4,7 +4,7 @@ import { toast } from "@/components/ui/use-toast";
 import CytoscapeComponent from 'react-cytoscapejs'
 import cytoscape from 'cytoscape';
 import fcose from 'cytoscape-fcose';
-import { useRef, useState } from "react";
+import { useEffect, useRef, useState } from "react";
```

Comment:
Where do you use useEffect in the code?

Reply:
I may have imported it for testing purposes, and now it needs to be removed-Done